File: src/3248.snake-in-matrix.3533/solution.java
class Solution {
    public int finalPositionOfSnake(int n, List<String> commands) {
        int i = 0;
        int j = 0;
        for(String command : commands){
            if(command.equals("UP")){
                i--;
            }else if(command.equals("RIGHT")){
                j++;
            }else if(command.equals("DOWN")){
                i++;
            }else {
                j--;
            }
        }
        return i*n+j;
    }
}

File: src/3248.snake-in-matrix.3533/solution.ts
function finalPositionOfSnake (n: number, commands: string[]): number {
    let x = 0;
    let y = 0;
    for (const command of commands) {
        if (command === 'LEFT') {
            y--;
        } else if (command === 'RIGHT') {
            y++;
        } else if (command === 'DOWN') {
            x++;
        } else {
            x--;
        }
    }
    return x * n + y;
}
